postgres_composite_types/__init__.py

```diff
@@ -52,31 +57,8 @@ class BaseField(models.Field):
 
     def db_type(self, connection):
         LOGGER.debug("db_type")
-
-        if connection.settings_dict['ENGINE'] != \
-                'django.db.backends.postgresql':
-            raise RuntimeError("Composite types are only available "
-                               "for postgres")
```

Comment:
Although could use PostgresDatabaseWrapper if you prefer. I copied this example out of the docs, but I thing your way below is cleaner.